src/js/franklin-dashboard.js

```diff
@@ -1,81 +1,95 @@
 'use strict';
 
+/* Dependencies */
 import angular from "angular";
 import packageJson from "../../package.json";
-import {
-    LoginComponent
-}
-from "./login/login.controller";
-
 require('angular-ui-router');
```

Comment:
We're mixing ES6 modules with browserify require statements. Lets make these all ES6.